File: app/src/main/java/com/a60n1/ejashkojme/models/Notification.java
package com.a60n1.ejashkojme.models;

import com.google.firebase.database.IgnoreExtraProperties;

@IgnoreExtraProperties
public class Notification {

    public String from;
    private String type;

    public Notification() {

    }

    public Notification(String from, String type) {
        this.from = from;
        this.type = type;
    }
}

File: app/src/main/java/com/a60n1/ejashkojme/models/Notification.kt
package com.a60n1.ejashkojme.models

import com.google.firebase.database.IgnoreExtraProperties

@IgnoreExtraProperties
class Notification {
    var from: String? = null
    private var type: String? = null

    @Suppress("unused")
    constructor()

    constructor(from: String?, type: String?) {
        this.from = from
        this.type = type
    }
}